source/whats-new.txt

```diff
@@ -22,6 +23,30 @@ Learn what's new by version:
 * :ref:`Version 1.1 <kafka-connector-whats-new-1.1>`
 * :ref:`Version 1.0 <kafka-connector-whats-new-1.0>`
 
+.. _kafka-connector-whats-new-1.8:
+
+What's New in 1.8
+-----------------
+
+- Updated the MongoDB Java driver dependency to version 4.7
+- Updated log messages in source and sink connectors. For a complete list
+  of log message updates, see
+  `KAFKA-302 <https://jira.mongodb.org/browse/KAFKA-302>`__ in JIRA. 
+- Added support for monitoring source and sink connectors with JMX.
+  To learn more about monitoring connectors, see the
+  :ref:`<kafka-monitoring>` page.
+
+Sink Connector
+~~~~~~~~~~~~~~
+
+- Added support in the MongoDB sink connector's Debezium change data
+  capture (CDC) handler for events produced by Debezium's MongoDB
+  change stream CDC handler. To learn more about Debezium's
+  MongoDB change stream CDC handler, see
+  `DBZ-4205 <https://issues.redhat.com/browse/DBZ-4205>`__
+  from RedHat. To learn more about the MongoDB sink connector's
+  Debezium CDC handler, see :ref:`<sink-fundamentals-cdc-handler>`.
```

Comment:
Issue:

I think there are a few items that make this less easy to read.

- Since CDC is only relevant to the sink connector, you could omit it. The possessive phrase is unnecessarily long.
- A term is less easy to read when an abbreviation "(CDC)" separates it. In this case, "handler" is separated from "Debezium's MongoDB change stream handler".
- Similar to the style guide recommendation of not using MongoDB names in the possessive form, avoid using possessives like "Debezium's" unless absolutely necessary.

Open for discussion, but I think that links in the What's New section need to link to very specific content that relates to what was described. If nothing like that exists, either we should create the ticket to provide that info in our docs, or it should be omitted. Here are the related issues:

- The RedHat JIRA ticket isn't a good resource to learn more about Debezium's MongoDB change stream CDC handler.
- The crosslink to sink-fundamentals-cdc-handler doesn't seem necessary since the page doesn't currently contain content directly related to the update. I would have expected to see a product page in the first link and how to configure it in the second link.

Suggestion:
Incorporating all the above:

```suggestion
- Added support for the Debezium MongoDB change stream CDC handler. You can now configure the connector to listen for events produced by this handler.
```

Reply:
Great edit. Was unsure how to make it clear the CDC handlers (Debezium's or MongoDB's) I was referring to throughout this item, but the second sentence makes it clear.

Thanks for the thoughtful suggestion 🚀